src/pimApi.js

```diff
@@ -71,14 +78,32 @@ function getOptionsFromParam(options, fnName) {
   }
 }
 
-function createArrayOfRequests(pimUrl, tableId, maxEntries, elements) {
+function createArrayOfRequests(pimUrl, tableId, maxEntries, elements, includeArchived) {
   const arr = [];
   for (let i = 0; i < elements - 1; i++) {
-    arr.push(`${pimUrl}/tables/${tableId}/rows?offset=${(i + 1) * maxEntries}&limit=${maxEntries}`);
+    const queryString = generateQueryString({ offset: (i + 1) * maxEntries, limit: maxEntries, archived: includeArchived });
+    const url = `${pimUrl}/tables/${tableId}/rows?${queryString}`;
+
+    arr.push(url);
   }
   return arr;
 }
 
+function generateQueryString({ offset = 0, limit, archived }) {
```

Comment:
[MINOR] Mit dem Request von Simon wäre es wahrscheinlich sogar nun ganz einfach:

```suggestion
const generateQueryString = (params) => _(params).omitBy(_.isNil).map((value, key) => `${key}=${value}`).join("&")
```

Reply:
Ja das stimmt. Da die Tests aber eine bestimmt Reihenfolge der querys erwarten, würd ich es lieber nicht ganz so allgemein definieren.